tx.go

```diff
@@ -71,7 +71,7 @@ type Tx struct {
 	id                     uint64
 	db                     *DB
 	writable               bool
-	pendingWrites          []*Entry
+	pendingWrites          map[string][]*Entry
```

Comment:
You can make this type as a special type just like the code below:
`type MyInt int`